b2share/modules/upgrade/upgrades/upgrade_2_1_7_to_3_0_0.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
```

Comment:
TODO: test that the upgrades really work from 2.1.7 to 3.0.0 

This has not been tested with existing database